src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -186,6 +186,7 @@ public partial class DataException : System.SystemException
         public DataException(string s) { }
         public DataException(string s, System.Exception innerException) { }
     }
+#nullable enable
```

Comment:
What's the plan for addressing the remainder of the types?

Reply:
@stephentoub am currently working on a separate PR to annotate all the rest of System.Data/System.Data.Common (based off of this one) - it should land in the next few days. The idea was to split:

* Provider abstractions which have almost no meaningful implementation, and which affect providers etc.
* All the rest, which is implementation-heavy and generally on the legacy/archives side.

In any case, we should soon have 100% of it annotated.

PS Two dependencies of System.Data which haven't yet been annotated are System.ComponentModel.TypeConverter and System.Xml.*. I'm leaving these areas nullable-disabled for now in my PR (but they're not huge).